src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -543,6 +564,10 @@ protected DynamoModel(IStartConfiguration config)
             InitializeNodeLibrary(preferences);
 
             LogWarningMessageEvents.LogWarningMessage += LogWarningMessage;
+
+            var renderPackageFactoryAsm = config.PackageManagerAddress ??
+                      AssemblyConfiguration.Instance.GetAppSetting("renderPackageFactoryAssembly");
```

Comment:
Ha! Again. A copy paste. That is actually the config file I'm looking for but I definitely shouldn't be checking PackageManagerAddress first.